### ISSUE TITLE
AM-softmax loss

### ISSUE THREAD

**Issue by sofpya** [KevinMusgrave/pytorch-metric-learning#726]
why don't have AM-softmax loss?

**Comment by KevinMusgrave** [comment]
Paper link: https://arxiv.org/abs/1801.05599
